docs_src/getting-started/tools/Ch-CommandLineInterface.md

```diff
@@ -0,0 +1,104 @@
+# Command Line Interface (CLI)
+
+As of the Hanoi release (v1.3.0), EdgeX Foundry provides a command line interface tool.
+
+## What is the EdgeX CLI?
+
+EdgeX-CLI is a command line interface that provides a convenient way for interacting with EdgeX Foundry micro services. It abstracts away all the underling infrastructure and allows the user to easily execute commands. It removes the need to manually track micro services parameters like host and port, as well as knowing the exact request path.
+
+It replaces the need to manually construct complex CURL commands. In particular, it helps with Post/Update requests (example: create a new event, or update a device) because the request body of these requests often include a lot of parameters. EdgeX-CLI provides an "interactive mode" that greatly simplifies the request construction process. Commands that represent Post/Update requests leverage the interactive mode, opening the user's default editor with preloaded request scaffolding ready to be customized. The interactive mode also works with in-line parameters - allowing in-line values of the CLI command to get loaded into the request scaffolding. 
+
+EdgeX-CLI is an easy to use helper tool that benefits both EdgeX developers and users. This is the first release of the CLI with future enhancements planned.  Currently, the CLI only works in insecure mode (that is without the EdgeX API gateway protecting EdgeX services), but there is a lot more to come in future EdgeX releases.
+
+## How to get the EdgeX CLI
+
+Download the appropriate compressed file for your operating system containing the edgex-cli executable from [GitHub](https://github.com/edgexfoundry/edgex-cli/releases).
+
```

Comment:
@jpwhitemn the original tar ball was missing the configuration.toml file, see issue https://github.com/edgexfoundry/edgex-cli/issues/353 
Should we add a sentence of two here to effect of creating
1) $HOME/edgex-cli
2) copying from the tarball the configuration.toml file from the res directory into the above $HOME/edge-cli directory?

Or does it make sense to add a script to do so?
@tonyespy and @difince  please weigh in.

Reply:
@mkbhanda - we are probably going to release a 1.3.1 of EdgeX (dot release of Hanoi).  Would it make sense just to fix this in the dot release (therefore leave the documentation as is)??

Reply:
I am fine with merging this doc as is. @difince mentioned that she had raised the install of configuration.toml back in August and on not hearing back from others forgotten about it. I too did not notice.